src/cascadia/TerminalApp/TerminalTab.cpp

```diff
@@ -274,7 +279,11 @@ namespace winrt::TerminalApp::implementation
     // - <none>
     void TerminalTab::SplitPane(SplitState splitType, const GUID& profile, TermControl& control)
     {
+        const auto activePaneId = _activePane->Id();

```

Comment:
perhaps a useful comment:

```suggestion
        // Make sure to take the ID before calling Split() - it will clear out the active pane's ID 
        const auto activePaneId = _activePane->Id();
```